### DISCUSSION TITLE
Spreading the word

### DISCUSSION THREAD

**Discussion by techntools** [Neoteroi/BlackSheep#359] [General]
https://asgi.readthedocs.io/en/latest/implementations.html#blacksheep

**Comment by RobertoPrevato** [comment]
Thank You, @techntools! Apologies for writing late.

**Comment by thomafred** [comment]
Some time ago, I also suggested blacksheep to the awesome-asyncio list:

https://github.com/timofurrer/awesome-asyncio/pull/106

The PR still needs votes :)
